app/_components/icons/IconMapType.tsx

```diff
@@ -0,0 +1,17 @@
+import BookmarkIcon from '@assets/svg/bookmarkIcon.svg';
+import LottoIcon from '@assets/svg/lottoIcon.svg';
+import ShopIcon from '@assets/svg/shopIcon.svg';
+import { PLACE_TYPE } from '@constants/map';
+
+const IconMapType = (placeType: string) => {
```

Comment:
`placeType`값이 `MAP_BUTTON_TYPE` 상수의 키 값만 들어오는 것 같은데 아래처럼 타입을 좁혀볼 수 있을 것 같아요!!
```suggestion
const IconMapType = (placeType: keyof typeof MAP_BUTTON_TYPE) => {
```

Reply:
감사합니다!!! 리뷰 달아주신 부분들 emotion 적용 후에 수정해볼게요